### PR TITLE
feat: Concise titles for the top nav

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

We discussed shorting the top nav item names. This would do that.

## Changes

* Changes the remaining longer text items to smaller ones

Screenshot 2023-11-23 at 15 13 01

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/layout/navigation-3000/navigationLogic.tsx [hunk 1]
```diff
@@ -321,13 +321,13 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
...
                             identifier: Scene.Events,
-                            label: 'Event explorer',
+                            label: 'Events',
                             icon: <IconLive />,
                             to: urls.events(),
                         },
                         {
                             identifier: Scene.DataManagement,
-                            label: 'Data management',
+                            label: 'Data',
```
Comment (Twixes) [inline frontend/src/layout/navigation-3000/navigationLogic.tsx:330]: I think the other changes are great, we can't go with this though. There's no data under "Data" here – that lives in "Events" and "People". "Data management" is just about the _shape_ of data. Hence I propose we rename it to the industry-standard "Taxonomy" (though we've got a lot of work left in that regard), and move it below both "People". Or just move while keeping "Data management".

Comment (andyvan-ph) [reply frontend/src/layout/navigation-3000/navigationLogic.tsx:330]: I like "Taxonomy".

If we were worried about too much change at once, we keep "Data management" for a while and move it as you suggest, then change it to Taxonomy at a later date, maybe alongside some improvements.

Comment (benjackwhite) [reply frontend/src/layout/navigation-3000/navigationLogic.tsx:330]: It's not just taxonomy though 🤔

Comment (andyvan-ph) [reply frontend/src/layout/navigation-3000/navigationLogic.tsx:330]: Moving away from "Data" would avoid confusion with "Data warehouse" down the line, too.

Comment (benjackwhite) [reply frontend/src/layout/navigation-3000/navigationLogic.tsx]: Tried something different (see screenshot)...

Comment (corywatilo) [reply frontend/src/layout/navigation-3000/navigationLogic.tsx]: What's the pragmatic approach here? I think data sufficiently gets the point across, even if there is technically data in other places.

**Review by Twixes — APPROVED** [review]
https://github.com/PostHog/posthog.com/pull/7227#discussion_r1407736854 just reminded me we should also turn "Project settings" into just "Settings" @benjackwhite 
Regardless, this PR makes pure sense

File: frontend/__snapshots__/posthog-3000-navigation--navigation-base.png
Comment (Twixes) [inline frontend/__snapshots__/posthog-3000-navigation--navigation-base.png:1]: Ah, forgot about breadcrumbs though: these need to be updated from both Homepage and Event Explorer

Comment (Twixes) [reply frontend/__snapshots__/posthog-3000-navigation--navigation-base.png:1]: Well, that was quick @benjackwhite Here's a follow-up PR idea then 😅

**Comment by corywatilo** [comment]
I don't think renaming "Event explorer" to "Events" is the right move here. "Events" makes me think it's where I can _manage_ events. I had "Live events" in some mocks, which I think is a little more clear that it's recent/near-realtime activity. But there's an important distinction between this and the actual Data management → Events section.

**Pull request merged**